src/app/app-routing.module.ts

```diff
@@ -1,7 +1,46 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { WishlistPageComponent } from './components/wishlist-page/wishlist-page.component';
+import { CartPageComponent } from './components/cart-page/cart-page.component';
+import { NotFoundComponent } from './components/not-found/not-found.component';
+import { MainPageComponent } from './components/main-page/main-page.component';
+import { ProductPageComponent } from './components/product-page/product-page.component';
+import { NightGuard } from './core/guards/night-guard.service';
+import { ProductResolver } from './core/resolvers/product.resolver';
 
-const routes: Routes = [];
+const routes: Routes = [
+  {
+    path: '',
+    component: MainPageComponent
+  },
+  {
+    redirectTo: ''
```

Comment:
this line causes an error:  `Invalid configuration of route` and app doesn't starting